### PR TITLE
Adding Remnant content by @Zitchas

### PR DESCRIPTION
**Pull request by MCOfficer**
Ref: endless-sky/endless-sky#3761

> 
> 
> This request expands the Remnant by adding one new ship, two outfits, and several sets of missions.
> 
> New ship:
> 
>     * The Pelican: Heavy freighter (sprite courtesy of Brick, and the majority of the stats as well)
> 
>     * Korath Hunter (removed due to lack of communication with the creator and no access to required files)
> 
> 
> New outfit:
> 
>     * Scanner Software (Is a very short range multi-purpose scanner that costs heat & energy instead of mass. This outfit is intended to give players a low-cost way to taste the benefits of having the various scanners)
> 
> 
> Note regarding the Tuning rifle: It is included here for the purpose of the ships coming equipped with it. This is taken from (and completely identical to) Brick's PR: "Add a Remnant Hand to Hand weapon #3672" This has been removed to avoid problems in keeping this synced with Brick's PR. References to the rifle have been commented out.
> 
> Note regarding missions: This mission set is intended to be started with a mid tier human ship such as a corvette, and then using Remnant ships as they become available. As before, the initial missions need to be done to get the Remnant License, then from there they a mix of parallel mission sets with a few intersections.
> 
> Modifications:
> 
>     * Modifies the "Remnant Defence" set to face more reasonable foes
> 
>     * Modifies the "Remnant License" mission so that the player is no longer told that they've done everything available right now.
> 
> 
> New Mission sets:
> 
>     * Second Contact (optional mission to give players that mess up First Contact an explanation for the blood test, and the option to go back and co-operate.)
> 
>     * 2 x bounty missions (hunting Chasers and Hunters respectively)
> 
>     * 3 x tech discovery/fetch (Heavy Laser, Catalytic Ramscoop, & Plasma Canon)
> 
>     * 2 x return the samples (Relations with the Drak and Sprites
> 
>     * 2 x Learning Sign (learning the basics of Remnant sign language)
> 
>     * 6 x Remnant Salvage set (mix of recovering history, transport, combat. Unlocks the Albatross, Pelican, and basic salvage outfits)
> 
>     * 3 x Expanded Horizons Quarg (learning about the Quarg)
> 
>     * 3 x Follow up mission checking on the Terminus Researchers
> 
>     * 1 x "Busy Stranger" mission that just lets the player know the missions are done for now.

### CODE REVIEW

**Review by MCOfficer — COMMENTED** [review]
that logo probably will be merged, but it doesn't really belong to this PR..

**Comment by MCOfficer** [comment]
and since github refuses to let me edit my review, another comment - this also PRs the logo to the main repo, so that can't be what you wanted.

**Comment by MCOfficer** [comment]
and since github refuses to let me edit my review, another comment - this also PRs the logo to the main repo, so that can't be what you wanted.

**Comment by Zitchas** [comment]
You're right, MCO, that wasn't what I wanted. I was trying to make a new fork to PR just the new title logo to the community build, but it ended up in this fork instead. I left it there to avoid making even more of a mess of things.

Apparently making a fork of a fork of something I already have a fork of just sends me back to my original fork. Or I messed up, one of the two.

**Review by Zitchas — APPROVED** [review]

**Comment by Zitchas** [comment]
> 
> 
> This is next on my list of PRs to merge, but I want to take a good long read of it first, because I never actually gave much feedback. We're also going to need to deal with the multiple merge conflicts.

I'm excited to see your feedback. The conflicts themselves aren't too big. I seem to recall having dealt with a bunch before. Without even looking at them, I suspect that they are the addition to  Brick's entry in the copyright, and a conflict with the addition of the tuning rifle to the outfitter and Remnant ships. Which should merge just fine (I even already have tuning rifles added to the ships, but commented out, ready for when he got his rifle merged. I'm not sure about the Remnant.txt file conflict, though...

**Comment by MCOfficer** [comment]
> Apparently making a fork of a fork of something I already have a fork of just sends me back to my original fork. Or I messed up, one of the two.

you want to create a new branch on your existing fork, make your changes there and PR from that branch. i think i've told you already in the original PR, but it was a mistake to work on master in the first place, usually you want to create a new branch for every PR.

**Comment by Amazinite** [comment]
*You broke it.*

**Comment by Zitchas** [comment]
OK, as best I can tell I successfully reverted all those changes. The title-image is gone, and it is asking to resolve changes again, so I guess it worked.

File: data/remnant.txt [hunk 1]
```diff
@@ -34,6 +34,9 @@ fleet "Small Remnant"
 		"Albatross (Turret)"
 	variant 1
 		"Albatross (Heavy)"
+	variant 20
```
Comment (Amazinite) [inline data/remnant.txt]: These Pelican odds are exceptionally high. This will make almost 40% of all small fleets spawn a single Pelican. I think that this variant number should be lowered form 20 to 9.

Ship | Current | New | Proposed
-----|-------|-------|----
Starling | 87% | 53% | 67.5%
Albatross | 13% | 8% | 10%
Pelican | N/A | 39% | 22.5%

File: data/remnant.txt [hunk 2]
```diff
@@ -65,6 +68,15 @@ fleet "Large Remnant"
 	variant 2
 		"Albatross (Heavy)"
 		"Albatross (Turret)"
+	variant 10
+		"Pelican" 3
+		"Albatross" 1
+	variant 10
+		"Pelican" 2
+		"Starling" 3
+	variant 10
```
Comment (Amazinite) [inline data/remnant.txt:89]: 1) The Pelican has no variants. This is fine, but that doesn't mean that every fleet variant with a Pelican in it needs to have default variant escorts. Perhaps make the Albatross in the first Pelican fleet variant a heavy version, or one of the Starlings in the second a Thrasher while the other two are Snipers, or the Starlings in the last variant Hunters.
2) Again with the variant frequencies. I suggest you drop all these variant frequencies to 2.

Fleet | Current | New | Proposed
----|----|----|----
Alba + Star | 88% | 47% | 75%
Double Alba | 12% | 6% | 10%
Pelican Fleet | N/A | 47% | 15%

File: data/remnant.txt [hunk 3]
```diff
@@ -168,3 +210,6 @@ outfitter "Remnant"
 	"Fuel Pod"
 	"Cargo Scanner"
 	"Outfit Scanner"
+	"Scanner Software"
+#	"Tuning Rifle"
```
Comment (Amazinite) [inline data/remnant.txt]: Please remove all references to the tuning rifle, even as comments, as these are things that should've been/were handled by the tuning rifle PR.

File: data/remnant.txt [hunk 4]
```diff
@@ -131,17 +165,25 @@ phrase "remnant"
...
+		"Reborn"
+		"Ascending"
+		"Soaring"
+		"Enduring"
 
 
 
 shipyard "Remnant"
 	"Starling"
-	"Albatross"
```
Comment (Amazinite) [inline data/remnant.txt]: Reason? I'm only reviewing everything outside of the missions right now, but there needs to be a good reason why access to the Albatross is being removed and locked behind even more missions.

File: data/remnant.txt [hunk 5]
```diff
@@ -131,17 +165,25 @@ phrase "remnant"
...
 
 shipyard "Remnant Gascraft"
 	"Gascraft"
 
 outfitter "Remnant"
+	"EMP Torpedo Bay"
+	"EMP Torpedo"
```
Comment (Amazinite) [inline data/remnant.txt]: These changes were done in the master already EMPs, so they shouldn't be part of your PR. (i.e. remove them from the outfitter so they aren't changes done by your PR.)

File: data/remnant ships.txt [hunk 6]
```diff
@@ -211,7 +219,7 @@ ship "Albatross"
 	explode "large explosion" 40
 	explode "huge explosion" 10
 	"final explode" "final explosion large" 1
-	description "Once they left human space, it became apparent to the Remnant that they would need to build their own shipyards in order to defend themselves if they were discovered by the Alphas or any other unfriendly faction. Using new composite materials that they discovered, they built ships very different from anything seen in human space."
+	description "Once they left human space, it became apparent to the Remnant that they would need to build their own shipyards in order to defend themselves if they were discovered by the Alphas or any other unfriendly faction. Using new composite materials that they discovered, they built two new types of ships based on the relics they found: One to explore, and one to defend. The Albatross lacks the range or speed of the Starling, but makes up for it with durability."
```
Comment (Amazinite) [inline data/remnant ships.txt]: I'm not sure this description change is necessary.
1) The current description is fine as is.
2) This new description doesn't tell us anything new.
See endless-sky#3867 on making descriptions that just compare the ship or outfit in question to another.

File: data/remnant ships.txt [hunk 7]
```diff
@@ -312,13 +323,15 @@ ship "Gascraft"
 		"shields" 800
 		"hull" 1500
 		"required crew" 1
-		"bunks" 1
-		"mass" 80
+		"bunks" 2
```
Comment (Amazinite) [inline data/remnant ships.txt:187]: I'm guessing these bunk changes pertain to the story. If not, they're unnecessary.

File: data/remnant ships.txt [hunk 8]
```diff
@@ -339,3 +353,68 @@ ship "Gascraft"
...
+	gun -8 -135 "Inhibitor Cannon"
+	turret 0 -67 "Thrasher Turret"
+	turret 56 35 "Point Defense Turret"
+	turret -56 35 "Point Defense Turret"
+	explode "small explosion" 30
+	explode "medium explosion" 60
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large" 1
+	description "The Pelican is closely based on the original semi-organic ships the Remnant found in the Ember Wastes. While they are technically not alive, their internal mechanisms and method of construction bear such a close resemblance to living organisms that they are often considered as such. These ships serve as the backbone of Remnant salvage efforts, as well as their resource distribution network."
```
Comment (Amazinite) [inline data/remnant ships.txt]: Previous Remnant ship descriptions make no mention of the materials found (namely the self-healing ship hull) being organic in nature. If this is a revelation from the story, I'm unsure if it should be mentioned in this ship description.

File: data/remnant ships.txt [hunk 9]
```diff
@@ -339,3 +353,68 @@ ship "Gascraft"
...
+	attributes
+		category "Heavy Freighter"
+		licenses
+			Remnant
+		cost 34600000
+		"shields" 25000
+		"hull" 11000
+		"required crew" 14
+		"bunks" 46
+		"cargo space" 547
```
Comment (Amazinite) [inline data/remnant ships.txt]: I'm guessing I wasn't the one to balance this, because 547 tons of cargo space for a ship of this size (like half the volume of a Bulk Freighter, which holds 600 tons) is astonishingly large, even if it is alien tech. That's just what I noticed at a glance, but I'll take a more thorough look at the rest of the stats later.

Right now it seems to me as if the Pelican was at one point a bigger ship closer in size to the Albatross, but the sprite size was changed to be almost exactly between the Starling and the Albatross (but closer to the Starling) without changing the stats.

File: data/remnant outfits.txt [hunk 10]
```diff
@@ -342,7 +342,25 @@ outfit "Quantum Key Stone"
...
+# Scanner:
 
+outfit "Scanner Software"
+	category "Systems"
+	licenses
+		Remnant
+	cost 100000
+	thumbnail "outfit/scanner software"
+	"mass" 0
+	"outfit space" 0
```
Comment (Amazinite) [inline data/remnant outfits.txt]: No no *no no* **no no no**.
I see that this "outfit" has heat and energy gen (the `"energy consumption"` attribute should be used instead of a negative energy generation, see the Syndicate shield generators), but zero outfit space means that this can be stacked infinitely (at least as much as the ship's other systems will allow due to the detriments), allowing for *very* powerful scanning at the cost of only a small amount of heat and energy drain.

File: data/remnant outfits.txt [hunk 11]
```diff
@@ -342,7 +342,25 @@ outfit "Quantum Key Stone"
 	"quantum keystone" 1
 	description "This precious artifact attunes a ship's quantum oscillations in a way that allows it to travel through certain otherwise impassible wormholes in the Ember Waste. The stones are rare and valuable, because they can only be mined from one location in the Waste."
 
+# Scanner:
```
Comment (Amazinite) [inline data/remnant outfits.txt]: Remove this comment.

File: data/remnant outfits.txt [hunk 12]
```diff
@@ -342,7 +342,25 @@ outfit "Quantum Key Stone"
...
+	"outfit space" 0
+	"cargo scan power" 8
+	"cargo scan speed" 1
+	"outfit scan power" 8
+	"outfit scan speed" 1
+	"asteroid scan power" 8
+	"tactical scan power" 10
+	"heat generation" 1
+	"energy generation" -1
+	description "The Remnant are too few in number to support extensive ground-breaking research in all areas of technology, so they prefer to focus their efforts on a few areas of importance. To make up the deficiency in other areas, the Remnant rely on salvaging and reverse engineering technology from those around them. This software is a key part of this effort, as it allows any ship the capacity to do detailed close-range scans to determine what they want to target. Its range is much shorter than that of dedicated scanning equipment, but it means every Remnant ship can be on the lookout for new technology."
```
Comment (Amazinite) [inline data/remnant outfits.txt]: I feel like this description could be less wordy.
And given the current nature of this outfit having zero outfit space, perhaps these should actually be integrated stats of the Gascraft and Starling. Either that, or this needs to be an outfit with actual mass, which could result in needing to change around some outfits on ships.

File: data/remnant outfits.txt [hunk 13]
```diff
@@ -328,7 +328,7 @@ outfit "Emergency Ramscoop"
 	"mass" 4
 	"outfit space" -4
 	"ramscoop" 0.5
-	description "The Ember Waste is the sort of place where only a very foolish captain would travel without some sort of device for replenishing hyperspace fuel. This ramscoop is not particularly powerful, but it allows a ship to explore without fear of getting stranded."
+	description "The Ember Waste is the sort of place where only a very foolish captain would travel without some sort of device for replenishing hyperspace fuel. While this risk is non-existent in Remnant ships due to their 'gills' which are effectively built-in ramscoops, some captains prefer to speed up the process by adding additional capacity. For the rare captain braving the area in non-Remnant ships, this ramscoop is not particularly powerful, but it allows a ship to explore without fear of getting stranded."
```
Comment (Amazinite) [inline data/remnant outfits.txt:329]: I like the additional lore of the gill-like structures being built in ramscoops, but I'm not sure if it's necessary. 

Remnant ships already have a considerable amount of power baked into the hull, that being built in shield and hull repair that is rather efficient. The Starling also has built in cloaking, and I already mentioned possibly making the scanning software built in as well. At this point, Remnant ships are a little over loaded with built in attributes.

File: data/map.txt [hunk 14]
```diff
@@ -4379,6 +4379,7 @@ system Arculus
 	trade Plastic 498
 	fleet "Large Remnant" 1500
 	fleet "Small Remnant" 1000
+	fleet "Korath Raid" 30000
```
Comment (Amazinite) [inline data/map.txt]: Any particular reason for the various fleet spawn rate changes? I see that you doubled or even quadrupled the spawn change for some fleets, and added new fleet spawns to different systems. I haven't reviewed exactly where these systems are in game, but it seems to me like some of these might not be necessary given the fact that the Ember Waste is really suppose to seem like a mostly empty wasteland, and adding more fleet spawns could ruin that atmosphere.

**Review by Amazinite — COMMENTED** [review]
Note that I have yet to read any of the story missions and I have not read any of the comments on the original PR, so I might be repeating some stuff that others have pointed out. This review is just a look over of all the changes outside of remnant missions.txt. I'll handle the real meat of the PR some time over the next 24 hours.

Notes:
* Remove any lines that deal with the tuning rifle, since that isn't part of this PR and only helps lead to merge conflicts
* Some of the changes I saw seem somewhat unnecessary, namely description changes to existing ships/outfits
* Overall looks nice, I didn't notice any syntax errors, but the details need a little polishing, especially those Pelican stats

File: data/remnant.txt (hunk 1 above)
Comment (Zitchas) [reply data/remnant.txt]: I set the pelican fairly high as it is currently the only non-combat ship in the Remnant inventory, so it is probably the primary ship used for non-combat missions. In terms of lore, I picture anytime they ferry bulk quantities of anything between worlds, do salvage runs, collecting gases/asteroid minerals/mining from anywhere, etc it is all going to be via Pelican. 

Ideally, this would be differentiated by the addition of an additional smaller freighter to their inventory. As a side note, Brick has agreed to take a stab at making a smaller Remnant freighter, and when that gets added these numbers would be reduced down significantly as the smaller freighter would take over about 2/3 of the Pelican's appearances in small fleets. 

Would you accept leaving the values as is with a re-balance/reduction later on when the Remnant have an additional freighter?

File: data/map.txt (hunk 14 above)
Comment (Zitchas) [reply data/map.txt]: These were changed to make it a bit more threatening environment, intending to put more pressure on the Remnant. As in they aren't just facing the occasional random Korath anymore, they're actually under threat of frequent raids. 

That being said, I can change these values via missions (and/or mission triggered events), right? I'm thinking it might contribute better to the atmosphere to revert to the original values and have events trigger after the first few missions that bump up the Korath frequencies. That would give the attentive player a better feel for the sudden increase in threat.

File: data/remnant.txt (hunk 4 above)
Comment (Zitchas) [reply data/remnant.txt]: I moved the Albatross down the mission chain a little so that the player isn't suddenly jumping from "we don't know who you are and won't sell you even a basic ramscoop" to "We trust you enough to let you buy a Heavy Warship" in the span of 7 missions. That always bugged me that such a covert and fearful group would give a stranger access to their heavy warship  after barely meeting them.

One of the main changes in the original set of missions is that I don't want to force players into using heavy warships right off the bat. I changed the balance somewhat so that the missions will have a sense of progression and escalation. Being able to do the initial ones in a human light or medium warship, then the next set expecting they've probably upgraded to a a starling, although if they want something heavy there's all the human stuff and the Shield Beetles. Later on they'll get the Albatross and the Pelican. 

Planned: 
- Once the medium freighter is done, it would be available right off the bat with the Starling too. 
- Further down the mission string there will also be a Peregrine (Fast medium warship) and the Vulture (bigger & better heavy warship) in the mid and late campaign respectively.
- Tentatively also planning on the penguin (updated gascraft) for a side mission series in the mid-campaign

File: data/remnant.txt (hunk 1 above)
Comment (Amazinite) [reply data/remnant.txt]: The problem with these values as is is that they drastically depress the appearance of Albatrosses and Starlings, the actual combative ships of the Remnant. It doesn't seem wise to make Pelicans so common if they're unlikely to be defended due to how much they depress the appearance of combative ships. The proposed variant frequency of 9 means that about 3/4ths of all small fleets are combative in nature while 1/4th are non-combative, a balance I find more fitting than 3/5ths vs 2/5ths given the possible dangers of having undefended freighters.

File: data/map.txt (hunk 14 above)
Comment (Amazinite) [reply data/map.txt]: I can understand having Korath raids rarely appear in more systems (20k - 30k frequency), but like I said, some fleets appear twice or four times as often as previously, which is a *substantial* increase from the status quo. (e.g. Caeculus raids doubled, Peragenor and Segesta from no raids to rather frequent raids, Stercutus raids quadrupled.)

File: data/remnant.txt (hunk 1 above)
Comment (Zitchas) [reply data/remnant.txt]: OK, sounds good. I'll make the change.

Hmmm, as far as that goes, might it be good to remove the Pelicans entirely and add separate fleets that are exclusively freight oriented?

File: data/remnant outfits.txt (hunk 11 above)
Comment (Zitchas) [reply data/remnant outfits.txt]: Removed.

File: data/remnant.txt (hunk 3 above)
Comment (Zitchas) [reply data/remnant.txt]: All references to the tuning rifle have been removed, with the exception of the Pelican. Did you want it removed from that one too?

File: data/remnant.txt (hunk 5 above)
Comment (Zitchas) [reply data/remnant.txt]: I've removed all traces of the EMPs from my PR.

File: data/remnant outfits.txt (hunk 10 above)
Comment (Zitchas) [reply data/remnant outfits.txt]: 1) Replaced the negative energy generation with positive energy consumption.

2) Assuming that people are willing to tolerate a decent amount of permanent heat and energy drain, they'll be able to... what, exactly? Scan a ship's outfits at something more than point-blank range? See how many crew they have? These aren't exactly game breaking abilities, and may help people become more immersed in the game. 

Potentially more valuable is that it helps encourage people to branch out and try alternate strategies. We get direct feedback for shield and hull damage, but energy, fuel, and heat don't have any meaningful result until they actually cripple a ship. If players can see this happening without wasting space on a frivolous outfit, it may encourage people to try these things out.

3) That being said, Remnant are unlikely to value cargo scanning nearly as much as much, so I halved it (and reduced the outfits and asteroid scanning power by a quarter.)

File: data/remnant.txt (hunk 3 above)
Comment (Amazinite) [reply data/remnant.txt]: I'd say yes since the tuning rifle isn't merged to the master.

File: data/map.txt (hunk 14 above)
Comment (Zitchas) [reply data/map.txt]: I'll reset these to defaults, and work on some events to boost the frequencies at various points in the mission series. Don't hold the PR for them or anything, It might be more impactful if the frequency of Korath raids doesn't change until players are fully committed to helping the Remnant and they've gotten used to how quiet the Ember Wastes actually are. So these events will get added with my next mission set when the Korath really start ramping up.

File: data/remnant outfits.txt (hunk 10 above)
Comment (Amazinite) [reply data/remnant outfits.txt]: The value of scanning itself isn't what's in question, it's the value of this outfit when compared to other scanning outfits. When given the choice between 13 tons of various scanning outfit or a no-space scanning outfit that does everything at only the cost of some heat gen and energy consumption, the no space outfit is a no brainer.

Comment (Amazinite) [reply data/remnant outfits.txt]: And notably, MZ already [expressed concern](https://github.com/endless-sky/endless-sky/pull/3761#issuecomment-395977301) about this outfit in the PR.
>(And, I'm not sure if it makes sense to have outfits that are "software," instead of a physical item you install in the ship.)

Comment (MCOfficer) [reply data/remnant outfits.txt]: you could make a module out of it, which takes up 3 outfit space or so

File: data/map.txt (hunk 14 above)
Comment (Zitchas) [reply data/map.txt]: OK, this part is done. Any system that didn't have any Korath (or small Remnant) ships appearing in it originally has been set to 40000, which, if I'm understanding it correctly, means they should appear once in a while, but extremely rarely. Everywhere else has been reset to defaults.

File: data/remnant outfits.txt (hunk 10 above)
Comment (Zitchas) [reply data/remnant outfits.txt]: Well, ideally the player would be limited to buying  a single one. One piece of software, get limited ability to scan everything in exchange for heat+energy drain. It'd still be worth getting the dedicated modules for the massive increases in range. 

But anyway, in order to keep this moving along I've increased the mass to 3 and changed the scanning values to something close to the normal scanners. Heat and energy aside, these are definitely the best scanners per ton out there, but no-one will be able to make infinite stacks of them anymore. Also added them to all the Remnant ships, and adjusted space accordingly. 

With the exception of the gascraft. Due to MZ's desire to ensure that the gascraft never have enough space to equip a cloaking device, I edited the scanning abilities directly onto the gascraft instead. (this was one of the original reasons for the scanner being a 0 mass outfit)

File: data/remnant.txt (hunk 2 above)
Comment (Zitchas) [reply data/remnant.txt:89]: OK, I've adjusted the Pelican fleet variant down to your recommend frequencies.

File: data/remnant.txt (hunk 3 above)
Comment (Zitchas) [reply data/remnant.txt]: Alright, the tuning rifle is completely gone now.

File: data/remnant ships.txt (hunk 7 above)
Comment (Zitchas) [reply data/remnant ships.txt:187]: Yes. I have a whole new string of missions coming up for the gascraft. :)

And as an amusing side benefit, it means that people will be able to take James along to see the Voide Sprites and at least some of the Remnant missions (although not the ones I have planned, where an NPC will be making use of that spare bunk.)

File: data/remnant outfits.txt (hunk 13 above)
Comment (Zitchas) [reply data/remnant outfits.txt:329]: Well, that's part of what I'm going for. In expanding the Remnant as a faction, I don't want to create another group feverishly trying to get the toughest warships to go toe to toe with the biggest opponents available in a slugfest. It is kind of hard because there's really only so many ways to do combat, and most weapons are too fast to make variations on the gascraft vs archon contest. So these ships as a whole are ones that are intended to be good platforms for alternate combat strategies. Hit and run using cloaking and/or afterburners, for instance. 

Giving them extra outfit space and 1-3 extra emergency ramscoops would mean that much more stuff that could be sold to cram in more weapons, whereas extra fuel regen gives players more flexibility in how much they use alternate special things (that all seem to burn to fuel), but without the ability to convert that space into weaponry. At the end of the day, fuel regen isn't that powerful a thing, and really fits a group that spends all its time wandering around in an unpopulated wasteland. And it has a much lower impact than adding 4-12 tons of space.

As for the scanners, I'm thinking about it. In a way, I prefer to leave it as a 3 tons module that players can choose to have (and choose to add more of if they want) rather than have it built-in. But building it in would work too.

File: data/remnant.txt (hunk 5 above)
Comment (Amazinite) [reply data/remnant.txt]: Well now you've removed the EMPs entirely should the PR merge. All you needed to do was remove them from the outfitter because if the PR was merged then they'd be in the outfitter twice in two locations.

**Review by Zitchas — APPROVED** [review]

File: data/remnant.txt [hunk 15]
```diff
@@ -66,7 +67,46 @@ fleet "Large Remnant"
 		"Albatross (Heavy)"
 		"Albatross (Turret)"
 
-
+fleet "Remnant Transport"
```
Comment (Amazinite) [inline data/remnant.txt:69]: So all Pelican spawns are part of Remnant Transport now?

Fleet | Current 
----|----
Lone Gull | 46.7%
Lone Pelican | 13.3%
Gull + Pelican | 21.3%
Pelican + other escorts | 18.7%
. |
Total Gull chance | 68%
Total Pelican chance | 53.3%

Just posting current spawn rates for now.

File: data/map.txt [hunk 16]
```diff
@@ -4379,6 +4379,8 @@ system Arculus
 	trade Plastic 498
 	fleet "Large Remnant" 1500
 	fleet "Small Remnant" 1000
+	fleet "Remnant Transport" 800
+	fleet "Korath Raid" 40000
```
Comment (Amazinite) [inline data/map.txt:4380]: Gonna do something I haven't done before and not entirely sure how accurate it is;
Smaller numbers after the fleet means the higher chance the fleet appears, therefore adding the inverse of all fleets in a system should give you an idea of the fleet "density" in that system, i.e. a number that describes how likely a fleet will be in the system where lower numbers describe higher chances of seeing any given fleet. I think this number can basically be looked at as the fleet frequency of the entire system.
Doing that, here are the numbers that I get for these systems.

System | Current "Density" | PR "Density" | % Diff
------|-------|------|-------
Arculus | 600 | 340 | 43.3%
Cinxia | 506 | 310 | 38.7%
Edusa | 1200 | 538 | 55.2%
Farinus | 3938 | 2639 | 33.0%
Pantica | 412 | 270 | 34.4%
Peragenor | N/A | 40000 | N/A
Perfica | 2791 | 2130 | 23.7%
Segesta | N/A | 20000 | N/A
Stercutus | 20000 | 13333 | 33.3%

So all Remnant systems will be more populous after this PR. Not sure that's the best result though given what I previously mentioned about the Ember Waste being better off seeming mostly empty. I'll propose new fleet frequencies in the future to get those % changes down. We'll need to fiddle with the preexisting fleet spawn rates.

File: data/remnant.txt (hunk 15 above)
Comment (Zitchas) [reply data/remnant.txt:69]: Yes, all Pelicans are part of Remnant Transport.

File: data/map.txt (hunk 16 above)
Comment (Zitchas) [reply data/map.txt:4380]: Yes, I'm planning on spending some time in game hanging out in those systems fluttering around in a gull seeing how the systems feel. I like having greater Remnant presence in their three "core" systems, as well as the pair of systems without planets that is only accessible through their core territory. (I consider those two systems to be part of their core territory, given proximity.) But the rest of the Ember wastes will probably need to see the frequencies decreased a bit to keep them feeling suitably empty.

Comment (jafdy) [reply data/map.txt:4380]: You maths is good @Amazinite. The frequency is the average time (with some units) between two of that fleet so the inverse is going to the proportional to pascal distribution frequency, because each fleet arrives independently you can simply add those frequencies up to get the frequency of any fleet arriving. Inversing that and you land back in the original units with a number for the average time between any fleet arriving.

In short, your maths does exactly what you think it does with accuracy as good you could want.

Comment (Amazinite) [reply data/map.txt:4380]: Time to make a spreadsheet that automates the process for me and let’s me compare systems with ease.

Comment (Amazinite) [reply data/map.txt:4380]: I think these changes are okay.
* Home systems: the systems the Remnant inhabit, only get slightly more populous
* Close systems: The systems attached to the home systems, get a little more populous
* Edusa: The wormhole system closest to home, becomes nearly 30% more populous
* Peragenor: Where the wormhole leads, actually has Remnant ships entering the system now
* Far systems: Those systems past the wormhole, now have occasional Remnant patrols, transports don't go as far.

image

I'd also suggest having small and large Remnant fleets spawn in Caeculus at about 30k.

File: data/remnant ships.txt [hunk 17]
```diff
@@ -214,7 +214,7 @@ ship "Gull"
 	explode "small explosion" 20
 	explode "medium explosion" 10
 	"final explode" "final explosion small" 1
-	description "While the starling was the first successful ship to use the technology derived from the alien ruins, the Gull is the most recent. The recent increase in Korath activities has caused foraging for resources to become a risky endeavor, forcing the Remnant to develop a specialized freighter for hauling cargo from remote sites without revealing their presence to hostile aliens."
+	description "While the starling was the first successful ship to use technology derived from alien ruins, the Gull is the most recent. The recent increase in Korath activities has caused foraging for resources to become a risky endeavor, forcing the Remnant to develop a specialized freighter for hauling cargo from remote sites without revealing their presence to hostile aliens."
```
Comment (Lorantine) [inline data/remnant ships.txt]: capital s in starling

Comment (Zitchas) [reply data/remnant ships.txt]: Thanks, corrected.

File: data/map.txt (hunk 16 above)
Comment (Zitchas) [reply data/map.txt:4380]: Caeculus has small and large at 30k.

spawn rates adjusted to match the above chart.

File: data/remnant outfits.txt (hunk 12 above)
Comment (Zitchas) [reply data/remnant outfits.txt]: Description was changed a while ago; but also no longer relevant as the ability to scan got merged into the ships pending a new outfit graphic.

File: data/remnant ships.txt (hunk 8 above)
Comment (Zitchas) [reply data/remnant ships.txt]: Viminal spaceport description: 

> "The spaceport is an enormous dome, built of the same resilient and **semi-organic material** as the hulls of Remnant ships. An opening at one end of the dome allows ships to fly in and out. Inside, the air is still cold, but at least you are sheltered from the violent winds that sweep across the rest of the planet's surface. Some of the locals, accustomed to the cold, walk about in their shirtsleeves as if this were a balmy summer day."
